### PR TITLE
Credential values json serialize/deserialize

### PR DESCRIPTION
**Pull request by kgoncharov**
* added cred vals serialization methods
* added corresponding tests

### CODE REVIEW

**Review by brentzundel — APPROVED** [review]

File: libursa/src/ffi/cl/mod.rs [hunk 1]
```diff
@@ -740,6 +740,114 @@ pub extern "C" fn ursa_cl_credential_values_free(credential_values: *const c_voi
...
+                "ursa_cl_credential_values_from_json: credential_values: {:?}",
+                credential_values
+            );
+            unsafe {
+                *credential_values_p = Box::into_raw(Box::new(credential_values)) as *const c_void;
+                trace!(
+                    "ursa_cl_credential_values_from_json: *credential_values_p: {:?}",
+                    *credential_values_p
+                );
+            }
```
Comment (blu3beri) [inline libursa/src/ffi/cl/mod.rs:836]:
```suggestion

            unsafe {*credential_values_p = Box::into_raw(Box::new(credential_values)) as *const c_void; }
            trace!(
                "ursa_cl_credential_values_from_json: *credential_values_p: {:?}",
                credential_values_p
            );

```

File: libursa/src/ffi/cl/mod.rs [hunk 2]
```diff
@@ -740,6 +740,114 @@ pub extern "C" fn ursa_cl_credential_values_free(credential_values: *const c_voi
...
+                "ursa_cl_credential_values_to_json: credential_values_json: {:?}",
+                credential_values_json
+            );
+            unsafe {
+                let credential_values_json = string_to_cstring(credential_values_json);
+                *credential_values_json_p = credential_values_json.into_raw();
+
+                trace!(
+                    "ursa_cl_credential_values_to_json: credential_values_json_p: {:?}",
+                    *credential_values_json_p
```
Comment (blu3beri) [inline libursa/src/ffi/cl/mod.rs:784]:
```suggestion
                    credential_values_json_p
```

**Comment by kgoncharov** [comment]
@blu3beri @ryjones 
kindly asking for an approval
Thanks in advance!

**Comment by blu3beri** [comment]
> @blu3beri @ryjones 
> 
> kindly asking for an approval
> 
> Thanks in advance!

I am not a maintainer. But maybe someone can approve on my behalf.

**Pull request merged**